src/element.ts

```diff
@@ -347,6 +349,14 @@ export class Element {
     return this.context;
   }
 
+  /** Draw pointer rect. This allows mouse interaction with the element */
+  drawPointerRect() {
+    if (this.pointerRect) {
+      const bb = this.getBoundingBox();
+      this.context?.pointerRect(bb.getX(), bb.getY(), bb.getW(), bb.getH());  
```

Comment:
On line 354, pointerRect is a boolean. On line 356, pointerRect is a method. 

But to me, it sounds like a struct containing x,y,width,height. 

I think both should have better names.

Reply:
I'm fine with keeping pointerRect as a method in render context. 

I think the boolean should be named `shouldDrawPointerRect`.  It makes the conditional statement easier to understand.

Otherwise, this PR looks fine to me.

Reply:
changed